### ISSUE TITLE
VERGE Wallet Not Accepting Passphrase

### ISSUE THREAD

**Issue by UncleGoose** [vergecurrency/VERGE#778]
Hello!  Like several before me my Verge wallet is suddenly not accepting my passphrase as valid in order to unlock my wallet.  I believe I am using the latest version for Mac (Wraith Protocol Qt-4-7-2018), and running the latest version of High Sierra OS.  Per previous threads I tried entering the passphrase using the Debug console with <walletpassphrase "passphrase here" 600> and get a code-14 error.  I am quite confident I am entering the passphrase correctly.  I don't remember if I created a backup (although I did so after this issue arose--not sure how that works with wallet locked).  I do, however, have a recovery phrase.  I also have tried downloading an Electrum wallet in the hopes that I could use my recovery phrase to get my Verge back, but there was an error during the installation, so it would not install.  I would greatly appreciate any help.  Thanks!

# Issue Template

## Prerequisites

* [ ] Are you running the latest version?

## User Story:

* "I'm submitting a ... "
  - [ ] bug report
  - [ ] feature request
  - [X ] support request 
  - [ ] translation request 

## Steps to Reproduce (for bugs)

* Steps :
1.
2.
3.
4.

## Your Environment

* Verge Release:
* CPU:
* GPU:
* RAM:
* OS:
    - [ ] I'm using Mac
    - [ ] I'm using Linux
    - [ ] I'm using Mac

**Comment by Positivism** [comment]
None of the etherum wallets work with Sierra, your best bet is to recover using the passphrase on a windows machine/vm/server.

**Comment by jrsikors** [comment]
Couple suggestions:
  1)  Use single quotes while using walletpassphrase
`walletpassphrase 'passphrase here' 600`
  2)  Try using the passphrase in the wallet gui , not console, where you can just 'unlock wallet'.  Have seen weird cases in the past where it works in the GUI but not in the console using walletpassphrase.  If it does work in the GUI, then do a Change Passphrase immediately

**Comment by jrsikors** [comment]
And I'm also curious why you think you have a "recovery phrase" and where you got it from.  As Positivism pointed out, that's for Electrum, and I doubt you had Electrum if you are running on a Mac.

Have you tried using this "Recovery phrase" as your passphrase?

**Comment by justinvforvendetta** [comment]
please clarify, are you trying to use a seedphrase (that electrum generated) or a password that you created for the Qt wallet?

**Comment by Positivism** [comment]
He's trying to use the password he made but it's not working, he somehow managed to get a seedphrase but as he has a mac he cannot use electrum to recover it.

**Comment by UncleGoose** [comment]
Thanks Jeff S!  I gave your suggestions a try but still no lucks o far.  Still I appreciate the suggestions.



-----Original Message-----

**Comment by UncleGoose** [comment]
So, when I first got my wallet I was prompted to create a recovery phrase (so I made it up myself rather than being issued the 12 or 24 words).  This was in addition to my passphrase.   I have tried both phrases both in the normal GUI and in the debug console and no luck.  


Thanks again for your help!



-----Original Message-----

**Comment by jrsikors** [comment]
All the wallets are different:
 ~Android wallet
 ~QT wallet
 ~Electrum wallet

The only way you can recover addresses between the different addresses is with a private key.

Seeds / recovery phrases / passphrases don't work between different wallets.

So as justin pointed out -- you need to go re-install whatever wallet you initially set up.  It definitely doesn't sound like it was a QT wallet.

Did you do it on your computer or on your phone?  If phone, then Android.  If computer, it has to be Electrum... but you will need a Windows machine to install the Electrum wallet.

**Comment by UncleGoose** [comment]
OK, thanks.  Perhaps the wallet I first created was an electrum wallet, although somehow I ended up with a Qt installer file on my computer (maybe a later attempt?  I set it up a while ago and don't recall exactly how it went down.  I do remember being surprised after I installed the wallet that it was an on-line wallet rather than a cold storage.)  In any case, I appreciate your input.



-----Original Message-----
